draft-packages/form/docs/CheckboxGroup.stories.tsx

```diff
@@ -44,15 +44,18 @@ class CheckboxGroupExample extends React.Component<Props> {
 
 export default {
   title: `${CATEGORIES.components}/${SUB_CATEGORIES.form}/Checkbox Group`,
-  component: CheckboxGroup,
+  component: CheckboxField,
```

Comment:
Was this intentional?

Reply:
Yep - I thought it made sense that you'd be able to control the checkboxField Props e.g checked on and off. Whereas the checkBoxGroup kinda just acts as the wrapper + a label and id. 

If you know how to have both checkboxField and checkboxGroup as a component plz let me know

Reply:
Ah right, makes me think that we shouldn't have a separate set of stories for CheckboxGroup/RadioGroup

Reply:
It would be nice to be able to control the Group and the Field in one story, but 🤷 

I'm unsure which is best given we have to pick one. I think this is fine for now